pyintegration/deephaven2/time.py

```diff
@@ -0,0 +1,775 @@
+#
+#   Copyright (c) 2016-2021 Deephaven Data Labs and Patent Pending
+#
+""" This module defines functions for handling Deephaven date/time data. """
+
+import jpy
+
+from deephaven2 import DHError
+from deephaven2.constants import TimeZone
+from deephaven2.dtypes import DateTime, Period
+
+SECOND = 1000000000  #: One second in nanoseconds.
+MINUTE = 60 * SECOND  #: One minute in nanoseconds.
+HOUR = 60 * MINUTE  #: One hour in nanoseconds.
+DAY = 24 * HOUR  #: One day in nanoseconds.
+WEEK = 7 * DAY  #: One week in nanoseconds.
+YEAR = 52 * WEEK  #: One year in nanoseconds.
+
+_JDateTimeUtils = jpy.get_type("io.deephaven.time.DateTimeUtils")
+
+
+def to_datetime(s: str, quiet: bool = False):
+    """ Converts a datetime string to a DateTime object
+
+    Args:
+        s (str): in the form yyyy-MM-ddThh:mm:ss and with optional sub-seconds after an
+            optional decimal point, followed by a mandatory time zone character code
+        quiet (bool, optional): when True, if the datetime string can't be parsed, this function returns None, otherwise
+            it raises an exception; default is False
```

Comment:
```suggestion
        quiet (bool): when True, if the datetime string can't be parsed, this function returns None; otherwise
            it raises an exception.  The default is False
```
You haven't documented optional like this anywhere else in your code.